config/kubernetes-incubator/org.yaml

```diff
@@ -639,7 +639,6 @@ teams:
     - lavalamp
     - philips
     - piosz
-    - roberthbailey
```

Comment:
This will remove you as a member for k-incubator. Staying as a member shouldn't affect email notifications so just want to double check this. :)

Reply:
This removes me as a member of the group that owns the incubator org (which seems appropriate). I'm still a member of the org itself (line 137).

Reply:
Ah my bad, the github diff view confused me.